File: 2346-largest-3-same-digit-number-in-string/largest-3-same-digit-number-in-string.py
class Solution:
    def largestGoodInteger(self, num: str) -> str:
        large = ""
        for i in range(len(num)-2):
            if num[i] == num[i+1] == num[i+2]:
                if num[i]*3 > large:
                    large = num[i]*3
                        
        return large

File: 2346-largest-3-same-digit-number-in-string/largest-3-same-digit-number-in-string.cpp
class Solution {
public:
    string largestGoodInteger(string num) {
        string large = "";
        for (int i = 0; i <= num.size() - 3; i++) {
            if (num[i] == num[i+1] && num[i] == num[i+2]) {
                string temp(3, num[i]);
                if (temp > large) {
                    large = temp;
                }
            }
        }
        return large;
    }
};